src/Microsoft.Health.Fhir.Shared.Core/Features/Resources/Patch/FhirPathPatch/Operations/EOperationType.cs

```diff
@@ -0,0 +1,38 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace FhirPathPatch.Operations
+{
+    /// <summary>
+    /// An enumeration of the supported types of FHIR Path Patch operations.
+    /// </summary>
+    public enum EOperationType
```

Comment:
Perhaps `PatchOperationType` for consistency?

Reply:
Done, makes more sense. I didn't prefix `FhirPathPatch` on any of the Operation classes because all the code in `FhirPathPatch` is internal except for the builder and a helper used for testing.